src/services/replyService.js

```diff
@@ -15,36 +15,31 @@ const {
 } = require('../utils/constants');
 
 module.exports = {
-  broadcastAndStoreReply: function(authenticatedUserInstance, { postId, body, createdAt }) {
-    return PostModel.findById(postId)
-      .then(postRecord => {
-        const postAuthor = postRecord.userId;
-        const postPermLink = postRecord.permLink;
-        const permLink = this.replyPermlink(postAuthor, postPermLink,
-          authenticatedUserInstance.username, createdAt)
-        return authenticatedUserInstance.broadcastReply({ postAuthor, postPermLink, permLink, body });
-      })
-      .then(broadcastSuccess => {
-        if (broadcastSuccesss) {
-          return authenticatedUserInstance.userInOurDb;
-        }
-      })
-      .then(userRecord => {
-        const permLink = this.replyPermlink(postAuthor, postPermLink,
+  broadcastAndStoreReply: function(authenticatedUserInstance, postPermLink, postAuthor, body) {
```

Comment:
We should keep the params the way it was. This new update means the dev will have to be accurate with the ordering of the params, introducing more potential  for mistakes and bugs.
```
function (user, reply) {
   const { createdAt, body, postId } = reply; // Deconstructing on separate line. Deconstructing a param in a function with multiple params makes it harder to read so it's generally discouraged
   // Do stuff
}
```

This way, the dev just passes in the user & reply, rather than trying to remember what order the params are received in